packages/autocomplete-preset-algolia/src/search/getAlgoliaHits.ts

```diff
@@ -2,11 +2,11 @@ import { Hit } from '@algolia/client-search';
 
 import { search, SearchParams } from './search';
 
-export function getAlgoliaHits<THit>({
+export function getAlgoliaHits<TRecord>({
```

Comment:
what happened to the idea of making this an indexName indexed object of records? The client doesn't have the feature, but I think we could assert the type here?

Reply:
I think it makes more sense to implement this type mapping in the client first, and then to leverage that in Autocomplete. It'll avoid shipping any inconsistencies.

(The more I work on multi-index UIs with the search client, the more I need this type support btw 🙂)